create-release/package-mainnet/example-client.js

```diff
@@ -41,64 +41,59 @@ if (test_data.length != KEY_SIZE) {
 }
 
 // Prepare request
-let signature = sign(request);
+let encrypted_request = encrypt(request);
 let post_data = JSON.stringify({
-    request: request,
-    signature: [...signature],
+    payload: [...encrypted_request],
 });
-
 // Send request to server
 let req = http.request({
     host: public_mirror_host,
     port: public_mirror_port,
-    path: '/signed-request',
-	method: 'POST',
-	headers: {
-	    'Content-Type': 'application/json',
-	    'Content-Length': Buffer.byteLength(post_data)
-	}
-}, (response) => {
-    	let buf = []
-        response.on('data', (chunk) => {
-            buf.push(chunk)
-        });
-
-        response.on('end', () => {
-            if(response.statusCode == 200) {
-                let result = decrypt(Buffer.concat(buf)).toString();
-                console.log(result);
-            } else {
-                console.log(`Http error, status: ${response.statusCode}: ${buf}`)
-            }
-        });
-
-        response.on('error', (error) => {
-            console.log('error occured while reading response:', error);
-        })
+    timeout: 120000,
+    path: '/encrypted-request',
+    method: 'POST',
+    headers: {
+        'Content-Type': 'application/json',
+        'Content-Length': Buffer.byteLength(post_data)
     }
+}, (response) => {
+    let buf = []
+    response.on('data', (chunk) => {
+        buf.push(chunk)
+    });
+
+    response.on('end', () => {
+        if (response.statusCode == 200) {
+            let result = decrypt(Buffer.concat(buf)).toString();
+            console.log(result);
+        } else {
+            console.log(`Http error, status: ${response.statusCode}: ${buf}`)
+        }
+    });
+
+    response.on('error', (error) => {
+        console.log('error occured while reading response:', error);
+    })
+}
 )
 req.write(post_data)
 req.end()
 
 // Crypto utilities
 function encrypt(buf) {
     let res = [];
-    let pub_key = crypto.createPublicKey(key);
 
-    // Each encrypted chunk must be no longer than the length of the public modulus minus (2 + 2*hash.size()).
-    // Since hash size is 32 (SHA256), this equals to 66.
-    const MAX_CHUNK_SIZE = KEY_SIZE - 66;
+    // Each encrypted chunk must be no longer than the length of the public modulus minus padding size.
+    // PKCS1 is 11 bytes of padding.
```

Comment:
Maybe we could point to here from where the const const PKCS1_PADDING_LEN is defined or use the const here so that if that size ever changes, the max chunk size would change also.